sdk/core/Azure.Core/tests/RequestUriBuilderTest.cs

```diff
@@ -6,313 +6,313 @@
-            uriBuilder.AppendQuery("c", "d");
-
-            Assert.AreEqual("http://localhost/ab?query=value&c=d", uriBuilder.ToUri().ToString());
-        }
-
-        [Test]
-        public void AppendingPathWithSpecialCharacters()
-        {
-            var uriBuilder = new RequestUriBuilder();
-            uriBuilder.Reset(new Uri("http://localhost/"));
-            uriBuilder.AppendPath("/");
-            uriBuilder.AppendPath("~!@#$%^&*()_+");
-            uriBuilder.AppendPath("b/");
-
-            Assert.AreEqual("http://localhost/~%21%40%23%24%25%5E%26%2A%28%29_%2Bb%2F", uriBuilder.ToString());
-#if NETCOREAPP
-            Assert.AreEqual("http://localhost/~%21%40%23%24%25^%26%2A%28%29_%2Bb%2F", uriBuilder.ToUri().ToString());
-#else
-            Assert.AreEqual("http://localhost/~!%40%23%24%25^%26*()_%2Bb%2F", uriBuilder.ToUri().ToString());
-#endif
-        }
-
-        [Test]
-        public void AppendingQueryWithSpecialCharacters()
-        {
-            var uriBuilder = new RequestUriBuilder();
-            uriBuilder.Reset(new Uri("http://localhost/"));
-            uriBuilder.AppendQuery("a", "~!@#$%^&*()_+");
-
-            Assert.AreEqual("http://localhost/?a=~%21%40%23%24%25%5E%26%2A%28%29_%2B", uriBuilder.ToString());
-#if NETCOREAPP
-            Assert.AreEqual("http://localhost/?a=~%21%40%23%24%25^%26%2A%28%29_%2B", uriBuilder.ToUri().ToString());
-#else
-            Assert.AreEqual("http://localhost/?a=~!%40%23%24%25^%26*()_%2B", uriBuilder.ToUri().ToString());
-#endif
-        }
-
-        [TestCase("?a", "?a")]
-        [TestCase("?a=b", "?a=b")]
-        [TestCase("?a=b&", "?a=b&")]
-        [TestCase("?d=b&", "?d=*&")]
-        [TestCase("?d=a", "?d=*")]
-
-        [TestCase("?a=b&d", "?a=b&d")]
-        [TestCase("?a=b&d=1&", "?a=b&d=*&")]
-        [TestCase("?a=b&d=1&a1", "?a=b&d=*&a1")]
-        [TestCase("?a=b&d=1&a1=", "?a=b&d=*&a1=")]
-        [TestCase("?a=b&d=11&a1=&", "?a=b&d=*&a1=&")]
-        [TestCase("?d&d&d&", "?d&d&d&")]
-        [TestCase("?a&a&a&a", "?a&a&a&a")]
-        [TestCase("?&&&&&&&", "?&&&&&&&")]
-        [TestCase("?d", "?d")]
-        public void QueryIsSanitized(string input, string expected)
-        {
-            var uriBuilder = new RequestUriBuilder();
-            uriBuilder.Reset(new Uri("http://localhost/" + input));
-
-            Assert.AreEqual("http://localhost/" + expected, uriBuilder.ToString(new[]
-            {
-                "A",
-                "a1",
-                "a-2"
-            }, "*"));
-        }
-
-        [Test]
-        public void QueryIsSanitizedAppendQuery()
-        {
-            var uriBuilder = new RequestUriBuilder();
-            uriBuilder.Reset(new Uri("http://localhost/"));
-            uriBuilder.AppendQuery("a", "b");
-
-            Assert.AreEqual("http://localhost/?a=*", uriBuilder.ToString(new string[]
-            {
-            }, "*"));
-        }
-    }
+//    public class RequestUriBuilderTest
```

Comment:
Was it intentional to leave this commented out?

Reply:
No, I thought I've reverted this change. These tests have been messing up with test runner.